web-services/handler.js

```diff
@@ -99,18 +101,19 @@ module.exports.getAction = async (event, context) => {
     var tb2 = await getData(urlPath, params);     
   }catch (error){
     console.log();
+    hasError = true;
   } 
   //process and return data for chartdb endpoint 
   return {
-    statusCode: 200,
     headers:{
       'Access-Control-Allow-Origin': '*',
       'Access-Control-Allow-Credential': true,
     },
+    statusCode: hasError? 500 : 200,
     body: JSON.stringify({
-        created: processDB(tb1.data,'cc'),          
-        resolved: processDB(tb2.data,'rc'),     
-    }),            
+      created: processDB(tb1.data,'cc'),          
```

Comment:
This processing (both created and resolved) should be done up in the try/catch section since you are invoking a function to perform some work (which might fail). I suggest you move this to after you getData.

Reply:
Yes, I did it and will do a new update. I appreciate your suggestion look like I am working in a real project with value codes. Thank you